api/swagger-spec/swagger.yml

```diff
@@ -52,6 +52,26 @@ paths:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
+  /v3/doctor/{train}/{release}/{uuid}:
```

Comment:
We should include deis component versions (including workflow-manager) in the cluster metadata.